src/SFA.DAS.EAS.Employer_Financial.Database/Scripts/PreDeployment/AML-2671-PopulateNullApprenticeshipCourseNames.sql

```diff
@@ -0,0 +1,50 @@
+﻿BEGIN TRY
+BEGIN TRANSACTION
+
+DECLARE @Frameworks as table(
+        FrameworkCode int,
+        ProgrammeType int,
+        PathwayCode int,
+        PathwayName varchar(max),
+        ApprenticeshipCourseName varchar(max));
+
+INSERT INTO @Frameworks (FrameworkCode, ProgrammeType, PathwayCode, PathwayName, ApprenticeshipCourseName)
```

Comment:
All statements _should_ end with a semi-colon - in a future version it will be mandatory.
https://docs.microsoft.com/en-us/sql/t-sql/language-elements/transact-sql-syntax-conventions-transact-sql?view=sql-server-2017

Reply:
Suggested changes made.